src/main/webapp/index.html

```diff
@@ -20,7 +21,22 @@
 <body onload="registerServiceWorker(); createStateDropdownList();">
   <main>
     <div id="header">
-      <h1>Census Viz</h1>
+      <h1>Census Viz
+        <div class="tooltip">
+          <i class="material-icons" id="faq">help_outline</i>
```

Comment:
Maybe add a one line summary of what censusviz is

Reply:
This is a good idea! I will discuss our "elevator pitch" with Kyra and Olivia and add in the best one-sentence summary we can come up with.

Reply:
Done